### DISCUSSION TITLE
Activation of tracking only after user consent

### DISCUSSION THREAD

**Discussion by kaboume** [johannschopplich/nuxt-gtag#8] [General]
Good morning,
Thanks for this plugin.
I use it on a website in France, a country in which we need to ask the user's consent, via the display of a banner, to trace his navigation on the site.
I was wondering how by default not to activate the tracking and to be able to activate it only if the user clicks on the button of the banner.
Does the plugin allow this?
THANKS.

**Comment by johannschopplich** [comment]
This is a feature request.

**Comment by johannschopplich** [comment]
Hey there! The feature was implemented in v1. 🙌
